### ISSUE TITLE
missing tags?

### ISSUE THREAD

**Issue by gelisam** [haskell/actions#4]
I tried [the `minimal` example](https://github.com/haskell/actions/tree/main/setup#usage) but [the action fails](https://github.com/gelisam/typelevel-rewrite-rules/pull/13/checks?check_run_id=1543202584) with

    Error: Unable to resolve action `haskell/actions@v1`, unable to find version `v1`

Since `actions/setup-haskell` had [a bunch of tags of the form `v1`](https://github.com/actions/setup-haskell/tags) but this project doesn't, I am guessing that this project also needs some version tags?

**Comment by jared-w** [comment]
Yeah, this repo doesn't have tags yet. I still need to finish verifying that everything works and I've been meaning to do so but it's been a long week. I'm dedicating a few hours tomorrow to really getting this repo fully usable.

(In the meantime, using something like `haskell/actions/setup@main` would work. Or any valid git ref)

**Comment by jared-w** [comment]
I didn't realize that the actions/setup-haskell repo had already been archived yesterday. That's what I get for not paying attention to github notifications on a Friday, I suppose... Anyway, I stayed up a little bit to get this done now given that it's already suggesting people use this project.

As a bonus, I got all of the rest of the outstanding issues cleaned up while I was at it. I'm going to close out the issue as this should hopefully be fixed, now.
